topSellCheapestGameTest.js

```diff
@@ -0,0 +1,29 @@
+import GenrePage from './pageGenre.js';
+import StartPage from './pageStart';
+import AgePage from './pageAge';
+import Url from './config';
+import NamePriceCheck from './utils.js';
+import Game from './pageGame.js';
+
+const url = new Url();
+const startPage = new StartPage();
+const agePage = new AgePage();
+const genrePage = new GenrePage();
+const utils = new NamePriceCheck();
+const game = new Game();
+
+fixture`Steam`
+    .page(url.steam);
+
+test('Top Sell Cheapest Game', async t => {
+    await startPage.themesClicker(startPage.themeTitle);
+    await genrePage.topSellClicker();
+    const priceFirstGame = await genrePage.getGame();
```

Comment:
not only game price and name should be checked. but also the initial price of the game and the discount